app/react/Markdown/components/specs/EntityData.spec.tsx

```diff
@@ -73,26 +72,4 @@ describe('EntityData Markdown', () => {
       expect(screen.getByText('Main Image translated')).toBeInTheDocument();
     });
   });
-
-  describe('Error handling (until upstream implementation is implemented)', () => {
```

Comment:
By deleting these specs we are losing the coverage for the error handling. Is it possible to test the dispatch of the warning instead?